src/main/java/com/google/devtools/build/lib/remote/SimpleBlobStoreFactory.java

```diff
@@ -36,7 +36,29 @@
 
   private SimpleBlobStoreFactory() {}
 
-  public static SimpleBlobStore createRest(RemoteOptions options, Credentials creds) {
+  public static SimpleBlobStore create(
+      RemoteOptions options, @Nullable Credentials creds, @Nullable Path workingDirectory)
+      throws IOException {
+
+    if (isRestUrlOptions(options) && isDiskCache(options)) {
```

Comment:
while you are at it, rename all occurrences of "Rest" to http?

Reply:
ok :)